spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaSecurityAutoConfiguration.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.spring;
+
+import static com.linecorp.armeria.internal.spring.ArmeriaConfigurationUtil.configurePorts;
+
+import java.net.InetSocketAddress;
+import java.util.Optional;
+
+import javax.annotation.Nullable;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.PropertySource;
+import org.springframework.util.SocketUtils;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.SessionProtocol;
+import com.linecorp.armeria.server.DecoratingServiceBindingBuilder;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.spring.ArmeriaSettings.Port;
+
+/**
+ * Spring Boot {@link Configuration} that provides Armeria integration.
+ */
+@Configuration
+@EnableConfigurationProperties(ArmeriaSettings.class)
+public class ArmeriaSecurityAutoConfiguration {
+
+    private static final Port DEFAULT_SECURITY_PORT = new Port().setPort(8001)
+                                                                .setProtocol(SessionProtocol.HTTP);
+
+    @Bean
+    ArmeriaServerConfigurator secureArmeriaServerConfig(ArmeriaSettings settings,
+                                                        ConfigurableEnvironment environment,
+                                                        @Value("${management.server.port:#{null}}")
+                                                        Integer actuatorPort,
+                                                        @Value("${management.endpoints.web.base-path:#{null}}")
+                                                        String actuatorBasePath) {
```

Comment:
I fetched Spring Actuator related fields to [solve what I mentioned](https://github.com/line/armeria/pull/2502#issuecomment-631326462). I am not sure if this is a good way. what do you think?

Reply:
I think it's OK. If there's any issue with this, we can fix it later.